### PR TITLE
Fixed: options.open.{appName, callback}

### LINKED ISSUE #229: Bug: `open.appName` doesn't work in v1.0.2
[`open.appName`](https://github.com/gruntjs/grunt-contrib-connect#open) doesn't work - setting it to any value (e.g. 'firefox', 'google chrome', ... on Mac OS) doesn't open the specified browser. This is due to the incorrect parameter being passed to opn, which replaced the open package in v1.0.2.

**Background**
opn was updated from v1.0.0 to opn 4.0.0 in [v1.0.2 of connect](https://github.com/gruntjs/grunt-contrib-connect/commit/01c420ce154805f0b0d632e0169403dffc5296bf)

opn v1.0.0 allowed a second string parameter to be set - the app to open the target with

opn v^4.0.0, however, only takes [1 optional second parameter](https://github.com/sindresorhus/opn#options). Only 2 values can be set in the config: `wait` and `app`.

### PR DESCRIPTION
**Pull request by bitnot**
Fixes #229 by passing open:appName config property to opn and calling open:callback.

### CODE REVIEW

**Comment by bitnot** [comment]
@vladikoff could you kindly take a look?

**Comment by intih** [comment]
Good stuff 👍 Would like to see this merged as `open.appName` not working is an issue for me.

A suggestion for this PR: is it worth updating the [readme](https://github.com/gruntjs/grunt-contrib-connect#open) to point to [opn](https://www.npmjs.com/package/opn) instead of [open](https://www.npmjs.com/package/open)? 

Also, `opn` allows you to [specify a string or array for `options.app` - array to pass arguments to the app](https://github.com/sindresorhus/opn#app) so the documentation should probably reflect that 😄

**Comment by XhmikosR** [comment]
This needs a rebase, no unrelated style changes and no unrelated version changes.

**Comment by bitnot** [comment]
Thanks for looking into this @XhmikosR!
Not sure if I should update the version and change log, please advise.

**Comment by XhmikosR** [comment]
@bitnot: no you shouldn't update any meta files, this is done when a new release is made.

File: docs/connect-options.md [hunk 1]
```diff
@@ -82,6 +82,7 @@ This can be one of the following:
 	  callback: function() {} // called when the app has opened
 	}
 	```
+Note that in [v0.9.0](https://github.com/gruntjs/grunt-contrib-connect/releases/tag/v0.9.0) [open](https://www.npmjs.com/package/open) was replaced with [opn](https://www.npmjs.com/package/opn) but the configuration remained the same for backwards compatibility. `targed`, `appName` and `callback` are the only supported keys in the config object.
```
Comment (XhmikosR) [inline docs/connect-options.md]: `targed` is this correct or it is a typo?

Comment (bitnot) [reply docs/connect-options.md]: Oh, thanks again! Fixed.

**Comment by XhmikosR** [comment]
@bitnot: can you skip README.md inclusion? We usually update it when we make the release.

**Comment by bitnot** [comment]
@XhmikosR
I was a bit concerned users may try passing `opn`-style config with `app` instead of `appName` and with unsupported `options`, so felt need to clarify in the ReadMe.
Thanks for merging!

**Comment by XhmikosR** [comment]
Will be in v2.0.0 when that is made.

Maybe you could add a test?

**Comment by bitnot** [comment]
I am not very good with node, but I will try to add a test.

Any release date in mind?

**Comment by XhmikosR** [comment]
It doesn't depend on my otherwise I'd made the release already. I don't have access to this npm package, so when https://github.com/gruntjs/grunt-contrib-connect/pull/252 is merged.

**Pull request merged**